backend/src/common/utils/hash.util.ts

```diff
@@ -0,0 +1,19 @@
+import crypto from 'crypto';
+import { promisify } from 'util';
+
+const scrypt = promisify(crypto.scrypt);
+
+const hash = async (password: string): Promise<string> => {
```

Comment:
We can use this `hash` util not only for passwords
```suggestion
const hash = async (value: string): Promise<string> => {
```